Cargo.toml

```diff
@@ -415,7 +415,7 @@ backtrace = { git = "https://github.com/rust-lang/backtrace-rs.git", rev = "7226
 color-eyre = { git = "https://github.com/eyre-rs/eyre.git", rev = "e5d92c3" }
 deltalake = { git = "https://github.com/delta-io/delta-rs", rev = "57795da" }
 ethnum = { git = "https://github.com/datafuse-extras/ethnum-rs", rev = "4cb05f1" }
-jsonb = { git = "https://github.com/databendlabs/jsonb", rev = "c7525d9" }
+jsonb = { git = "https://github.com/b41sh/jsonb", rev = "10462ca" }
```

Comment:
Why change to `b41sh/jsonb `?

Reply:
this is a tmp rev, will changed back after this PR merged  https://github.com/databendlabs/jsonb/pull/64